.github/workflows/go-e2e.yml

```
@@ -0,0 +1,112 @@
+name: Go End to End
```

```suggestion
name: End to End
```
nit. Language doesn't matter when tests are end-to-end ;)